### ISSUE TITLE
As a site owner, I can create a card block that does not crop the image in a predefined format

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#5376]
# Story

As a site owner, I can create a card that does not crop the image in a predefined format.

Currently you can only choose a square, widescreen, or circle image format.   

# Proposed solution

Add a "no crop" option that allows the card to use the original image aspect ratio.   This would be especially useful for images like book covers.   

<img width="1504" alt="Screen Shot 2022-06-15 at 11 57 36 AM" src="https://user-images.githubusercontent.com/1036433/173883749-d55f1ae3-53cc-4961-bfac-179c9b8dfbf3.png">

**Comment by pyrello** [comment]
This is very much needed. Also it is going to lead to a lot of very questionable looking cards ;)

**Comment by cory-skeers** [comment]
> This is very much needed. Also it is going to lead to a lot of very questionable looking cards ;)

This is equally necessary and terrifying...

**Comment by joewhitsitt** [comment]
it'd be nice to do some kind of limit on this feature, like to prevent a 200x900 px image. http://www.placecage.com/c/200/900

**Comment by pyrello** [comment]
> it'd be nice to do some kind of limit on this feature, like to prevent a 200x900 px image. http://www.placecage.com/c/200/900

I disagree with enforcing a limit because it adds complexity to the maintenance of the application for what are essentially edge cases. Getting people not to do things like the above should be the purview of good documentation and educational outreach, not code.
